vault/changelog.md

```diff
@@ -2,11 +2,35 @@
 id: 9bc92432-a24c-492b-b831-4d5378c1692b
 title: Changelog
 desc: ''
-updated: 1637083041974
+updated: 1637346660450
 created: 1601508213606
 nav_order: 2
 ---
 
+## 0.69.1
+
+### Features
+- feat(workspace): Initialize Workspace command can create native workspaces ([[docs|dendron.topic.workspace.native]]) (#1701) @kaan
+
+### Enhancements
+- enhance(markdown): expose `desc` frontmatter property for variable substitution (([[docs|dendron.topic.markdown#frontmatter-variable-substitution]]) (#1721) @hikchoi
+  - View the [difference](https://github.com/dendronhq/dendron-site/pull/284/files)
+- enhance(workspace): improved task note decorations @kaan
+  * Task note decorations are colored with the same color as the link to make them easier to distinguish from regular text. ([[docs|]])
```

Comment:
I'm not certain enough about how the decorators work in order to understand how I should update the docs.

Reply:
<img width="835" alt="user_kevin_journal_2021_11_19_md_—_dendron__Workspace_" src="https://user-images.githubusercontent.com/409321/142681385-9f3ad3a5-5884-4dfb-b80c-ab2d95f1ca12.png">

Reply:
for the inline items, can you add them to [[Tasks|dendron://dendron.dendron-site/dendron.topic.tasks]]?